pandas/compat/_optional.py

```diff
@@ -11,7 +13,7 @@
     "fsspec": "0.7.4",
     "fastparquet": "0.3.2",
     "gcsfs": "0.6.0",
-    "lxml.etree": "4.3.0",
+    "lxml": "4.3.0",
```

Comment:
is this intentional or a merge error (looks unrelated)

Reply:
my bad.